applications/nrf_desktop/sample.yaml

```
@@ -1,15 +1,34 @@
 sample:
   name: nRF Desktop
   description: HID device reference design
+common:
+  sysbuild: true
+  tags: bluetooth ci_build sysbuild
+  harness: console
+  harness_config:
+    type: multi_line
+    ordered: false
+    regex:
+      - "app_event_manager: e:module_state_event module:main state:READY"
+      - "ble_state: Bluetooth initialized"
+      - "settings_loader: Settings loaded"
+      - "ble_bond: Selected BLE peers"
+      - "(ble_adv: Advertising started)|(ble_scan: Scan started)"
+      - "dfu: Secondary image slot is clean"
 tests:
+  applications.nrf_desktop.zdebug.uart:
```

With an intention of adding other DKs to this test option (once we make DKs run by default with UART)

```suggestion
  applications.nrf_desktop.dk.zdebug.uart:
```

---

I prefer not to include `dk` infix in the test name. Actually any board that provides logs over UART could be added to this group.